charts/actions-runner-controller/templates/controller.metrics.serviceMonitor.yaml

```
@@ -4,6 +4,9 @@ kind: ServiceMonitor
 metadata:
   labels:
     {{- include "actions-runner-controller.labels" . | nindent 4 }}
+  {{- with .Values.metrics.serviceMonitorLabels }}
+    {{- toYaml . | nindent 8 }}
+  {{- end }}
```

Hey! Thanks for the PR. Just confirming, but how did you test it?

I think this doesn't work as it removes all the preceding spaces until the end of line 6, resulting in something like the below for e.g. `serviceMonitorLabels: {"foo": "FOO", "bar": "BAR"}`:

```
metadata:
  labels:
    release: whatever
    chart: actions-runner-controllerfoo: FOO
        bar: BAR
```

I guess it should better be:

```suggestion
  {{- with .Values.metrics.serviceMonitorLabels }}
    {{ toYaml . | nindent 4 }}
  {{- end }}
```

so that the prev example becomes:

```
metadata:
  labels:
    release: whatever
    chart: actions-runner-controller
    foo: FOO
    bar: BAR
```

Would you mind giving it a shot once again?

---

Well I've been misunderstanding how nindent works. So `nindent` seems to add an empty line at the beginning so in the current form the above example would become:

```
metadata:
  labels:
    release: whatever
    chart: actions-runner-controller
        foo: FOO
        bar: BAR
```

Then perhaps this should be:

```suggestion
  {{- with .Values.metrics.serviceMonitorLabels }}
    {{- toYaml . | nindent 4 }}
  {{- end }}
```

---

@mumoshu fixed, yes, I missed the number of spaces.